OpenRA.Game/Settings.cs

```diff
@@ -269,6 +269,12 @@ public class KeySettings
 		public Hotkey SupportPower05Key = new Hotkey(Keycode.UNKNOWN, Modifiers.None);
 		public Hotkey SupportPower06Key = new Hotkey(Keycode.UNKNOWN, Modifiers.None);
 
+		public Hotkey PauseAltKey = new Hotkey(Keycode.SPACE, Modifiers.None);
```

Comment:
Alright, I will tweak it around a littlebit. Spacebar is usually related to play/pause, but I can change it to something else, no problem with that.